lib/v2/screens/authentication/recover/recover_account_search/interactor/mappers/fetch_account_info_state_mapper.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/i18n/import_key/import_key.i18n.dart';
+import 'package:seeds/v2/screens/authentication/recover/recover_account_search/interactor/viewmodels/recover_account_state.dart';
+
+class FetchAccountInfoStateMapper extends StateMapper {
+  RecoverAccountState mapResultToState(RecoverAccountState currentState, Result userInfo, String userName) {
+    if (userInfo.isError) {
+      return currentState.copyWith(
+          pageState: PageState.failure, errorMessage: "Error Loading Account".i18n, isValidAccount: false);
+    } else {
+      var accountInfo = userInfo.asValue!.value as MemberModel;
+      // ignore: unnecessary_null_comparison
+      if (accountInfo != null) {
```

Comment:
account info cannot be null since you said ! when it was being created, 
The null check is not needed.

Reply:
or make it nullable, then do the null check 

var accountInfo = userInfo.asValue?.value as MemberModel?;

Reply:
accountInfo can be null, I did your second suggestion. Thank you